napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -9,6 +9,8 @@
     QSlider,
 )
 
+from napari.utils.interactions import KEY_SYMBOLS
```

Comment:
Can this be a relative import. We use absolute imports in our tests, but otherwise relative ones (will then go with group below)

Reply:
Yes, thanks for pointing that out!  I have fixed it.  :)